app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.select(:id, :name, :username, :email).where(is_disabled: false).sort_by {|user| user.name}
```

Comment:
Could you move the `.where(is_disabled: false)` into a named scope?
Then this line can read: `User.active.select(...` which would be nicer.